component/otelcol/consumer.go

```diff
@@ -0,0 +1,19 @@
+package otelcol
+
+import (
+	otelconsumer "go.opentelemetry.io/collector/consumer"
+)
+
+// Consumer is a compbined OpenTelemetry Collector consumer which can consume
```

Comment:
```suggestion
// Consumer is a combined OpenTelemetry Collector consumer which can consume
```

Reply:
😞 sorry, one day I'll figure out how to get spell checking in Vim to be reliable. Right now I usually have it off because there's just way too many false positives.